capstone-app/src/app/modules/maps/map/map.component.spec.ts

```diff
@@ -27,29 +25,102 @@ describe('MapComponent', () => {
     expect(fixture.componentInstance).toBeTruthy();
   });
 
-  it('test making multiple markers', () => {
+  it('test multiple map marker click event', () => {
```

Comment:
Are these tests randomly failing occasionally like your test app was? If so, you may want to include a comment somewhere describing the issue. And maybe a TODO to fix.